src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +61,26 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// function to generate application details content
+const dynamicRowGeneration = (rowElementMaxCount, appDetailLabels, loading) => {
+  const labelArray = appDetailLabels.map(label => {
+    // only generate elements that are populated
+    if (label[0] && label[0]!='' && label[0].length > 0) {
```

Comment:
Is there a better way for me to check if a value should be included? I think just making sure there are no null, blank, or 0 length entries should catch everything. Another approach I was thinking of was using the "typeof" function to check each element:
`if(typeof label[0] == 'string') {
    if(label[0] != '') return generateLabelSpanContents()
}
`

Reply:
The check is even simpler with lodash's [isEmpty](https://lodash.com/docs/4.17.15#isEmpty) function.